client/index.html

```diff
@@ -26,7 +26,7 @@ <h1 class="Article__h1">
         <img src="./view/Badge/github_logo.svg" alt="Count of github stars" class="Badge__logo"> 10.5k
       </a>
       <a class="Badge" href="https://twitter.com/Browserslist">
-        <img src="./view/Badge/twitter_logo.svg" alt="Count of twitter followers" class="Badge__logo"> 4k
+        <img src="./view/Badge/twitter_logo.svg" alt="Count of twitter followers" class="Badge__logo Badge__logo--github"> 4k
```

Comment:
Let's write `Twitter` and `GitHub` instead of `twitter` and `github` in `alt=`

@igorlukanin done it in closed duplicated PR https://github.com/browserslist/browsersl.ist/pull/403